modules/core/04-channel/keeper/handshake.go

```diff
@@ -282,7 +282,7 @@ func (k Keeper) WriteOpenAckChannel(
 	channel.Counterparty.ChannelId = counterpartyChannelID
 	k.SetChannel(ctx, portID, channelID, channel)
 
-	k.Logger(ctx).Info("channel state updated", "port-id", portID, "channel-id", channelID, "previous-state", channel.State.String(), "new-state", types.OPEN.String())
+	k.Logger(ctx).Info("channel state updated", "port-id", portID, "channel-id", channelID, "previous-state", types.INIT.String(), "new-state", types.OPEN.String())
```

Comment:
actually this previous state shouldn't always be INIT from previously, though the previous `channel.State` was also incorrect as it would have been set to `OPEN` on line 280. maybe you should actually grab it from the channel on line 275.